3DCalendar/CalendarView.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  CalendarView.swift
+//  bolt
+//
+//  Created by Najia Haider on 2/20/19.
+//  Copyright © 2019 Najia Haider. All rights reserved.
+//
+
+import UIKit
+
+class CalendarView: UIView, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+ 
+    var numDaysPerMonth = [31,28,31,30,31,30,31,31,30,31,30,31]
```

Comment:
These can be `let`s

Reply:
we are modifying this! `//leap years
        if currentMonthIndex == 2 && currentYear % 4 == 0 {
            numDaysPerMonth[currentMonthIndex-1] = 29
        }`